content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -68,43 +69,60 @@ An example of an IPv6 CIDR: `fdXY:IJKL:MNOP:15::/64` (this shows the format but
 
 ## Services
 
-If your cluster has IPv4/IPv6 dual-stack networking enabled, you can create {{< glossary_tooltip text="Services" term_id="service" >}} with either an IPv4 or an IPv6 address. You can choose the address family for the Service's cluster IP by setting a field, `.spec.ipFamily`, on that Service.
-You can only set this field when creating a new Service. Setting the `.spec.ipFamily` field is optional and should only be used if you plan to enable IPv4 and IPv6 {{< glossary_tooltip text="Services" term_id="service" >}} and {{< glossary_tooltip text="Ingresses" term_id="ingress" >}} on your cluster. The configuration of this field not a requirement for [egress](#egress-traffic) traffic.
+If your cluster has dual-stack enabled, you can create {{< glossary_tooltip text="Services" term_id="service" >}} which can use IPv4, IPv6, or both. 
 
-{{< note >}}
-The default address family for your cluster is the address family of the first service cluster IP range configured via the `--service-cluster-ip-range` flag to the kube-controller-manager.
-{{< /note >}}
+Services will default to the address family of the first service cluster IP range configured  via the `--service-cluster-ip-range` flag to the kube-controller-manager.
 
-You can set `.spec.ipFamily` to either:
+If you want to explicitly specify which IP address is the primary IP address (which is the address assigned for the Service as `.spec.ClusterIP`), you can optionally use the `.spec.ipFamilyPolicy` field. This field accepts one of the following values:
```

Comment:
`s/IP address/IP address family/`

Not to be confused with user ability to select an actual IP for their services.

Reply:
We now say:
```
If your cluster has dual-stack enabled, you can create {{< glossary_tooltip text="Services" term_id="service" >}} which can use IPv4, IPv6, or both. 

The address family of a Service defaults to the address family of the first service cluster IP range (configured via the `--service-cluster-ip-range` flag to the kube-controller-manager).
```